### PR TITLE
create transaction

### PR DESCRIPTION
**Pull request by sudu-0-0-7**

### CODE REVIEW

File: src/FileCabinet/SuiteScripts/salesorder_record_cs.js [hunk 1]
```diff
@@ -4,24 +4,47 @@
...
-    function pageInit(){
+    /**
+     * @param{record} record
+     */
+    function (record) {
+        function pageInit() {
+
+        }
+
+        function completeTransaction(curtRecId, invoiceSearchResultCount, itemFulfillmentsearchResultCount) {
```
Comment (jampalu) [inline src/FileCabinet/SuiteScripts/salesorder_record_cs.js:15]: change the function name to "createTransactions" as we are actually creating not completing right?

**Review by jampalu — CHANGES_REQUESTED** [review]

File: src/FileCabinet/SuiteScripts/salesorder_record_ue.js [hunk 2]
```diff
@@ -2,79 +2,64 @@
...
-         * @since 2015.2
-         */
+    (record, curRec, search) => {
         const beforeLoad = (scriptContext) => {
             try {
-                var transactionSearchObj = search.create({
-                    type: "transaction",
-                    filters:
-                        [
-                            ["salesorder", "anyof", "51096"],
```
[the remark this answers is not in the document]
Comment (sudu-0-0-7) [reply src/FileCabinet/SuiteScripts/salesorder_record_ue.js:25]: Done!

File: src/FileCabinet/SuiteScripts/salesorder_record_ue.js [hunk 3]
```diff
@@ -2,79 +2,64 @@
...
-                        ],
-                    columns:
-                        [
-                            search.createColumn({ name: "salesorder", label: "Sales Order" })
-                        ]
-                });
-                var searchResultCount = transactionSearchObj.runPaged().count;
-                if (searchResultCount = 0) {
-                    if (scriptContext.type === 'view') {
+                if (scriptContext.type === 'view') {
```
[the remark this answers is not in the document]
Comment (sudu-0-0-7) [reply src/FileCabinet/SuiteScripts/salesorder_record_ue.js:12]: Done!

File: src/FileCabinet/SuiteScripts/salesorder_record_ue.js [hunk 4]
```diff
@@ -2,79 +2,64 @@
...
+                    var curtRecId = scriptContext.newRecord.id
```
[the remark this answers is not in the document]
Comment (sudu-0-0-7) [reply src/FileCabinet/SuiteScripts/salesorder_record_ue.js:13]: Done!

File: src/FileCabinet/SuiteScripts/salesorder_record_cs.js (hunk 1 above)
Comment (sudu-0-0-7) [reply src/FileCabinet/SuiteScripts/salesorder_record_cs.js:15]: Done!

**Pull request merged**